src/main/scala/Column.scala

```diff
@@ -336,7 +352,9 @@ object Column {
             if (node.isMissingNode) {
               None
             } else {
-              Some(node.get(propertyName).asText())
+              if (!node.path(propertyName).isMissingNode) {
```

Comment:
You could've used that `getMaybeNode` extension functions we wrote here.

Reply:
Done